GameData/000_FilterExtensions Configs/SubCategories_LBP.cfg

```diff
@@ -0,0 +1,56 @@
+SUBCATEGORY
+{
+	name = Modern Ships
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = title
+			value = modern,dg,medium,tri,tumble,vette
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = Carrier Parts
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = title
+			value = carrier
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = WWII Hulls
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = folder
+			value = LShipPartsWW2
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = Subs
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = folder
+			value = sub*
```

Comment:
Does this even work? No regex support and I don't remember OS's allowing '*' characters in file paths. I'd also be inclined to say "sub" isn't unique enough for a path prefix. I'd much rather a check against multiple paths than one that'll include false positives